### ISSUE TITLE
Could it block apps that have ROOT permission?

### ISSUE THREAD

**Issue by jeffgyf** [ukanth/afwall#988]

**Comment by bonbonboi** [comment]
Yes it does, but if that app with root permission just issued another iptables commands then it can bypass afwall rules because ifwall doesn't lock the iptables. That's far as I know. Correct me if I'm wrong.
